Resources/Prototypes/Maps/phobos.yml

```diff
@@ -0,0 +1,60 @@
+- type: gameMap
+  id: phobos
+  mapName: 'phobos'
+  mapPath: /Maps/phobos.yml
+  minPlayers: 0
+  maxPlayers: 50

```

Comment:
I'm no expert but min 0 max 50 seems like an... odd range? Was it intended like this?

Reply:
I just kinda picked a number. Any suggestions on making it higher/lower?

Reply:
I'm not really a mapper, but a map that is targeted for up to 50 players should have a minimum around 15-20 , judging by other map prototypes. Might want to work that out with experienced mappers in Cartography

Reply:
Map size updated to 65

Reply:
I repeat: the issue is that you have configured the map to be an equally valid choice (as far as the game engine is concerned) for 65 players or 4. Do you see the issue with putting 4 players on a map large enough for 65

Reply:
OK I understand the issue. I will count up the spawns and leave like 10 spots for wiggle room for greys?

Reply:
Look at a similar sized map to get an idea for individual role slot numbers, but specifically I meant that you need to set a `minPlayers` value so the map does not get picked if there are too few players in the lobby